packages/docs/docs/getting-started/create-unirep-app.md

```diff
@@ -44,112 +46,93 @@ See [hardhat tutorial](https://hardhat.org/hardhat-runner/docs/getting-started#q
 
-In a new terminal window, start a frontend server with the following command from root:
+The frontend is the **[user](../protocol/users-and-attesters.md#users-👤)**s' interface to interact with the attester (`UnirepApp.sol`).<br/>In a new terminal window, from root, run:
 
 ```shell
 yarn frontend start
 ```
 
 The frontend will be running at `http://127.0.0.1:3000/` by default.
 
-### Create-Unirep-App Architecture
-
+:::info
+##  🏗️  `create-unirep-app` Architecture
 
-It is important to note the architecture of this application is different than traditional dapps. Traditionally, dapps have to force users to use a browser wallet, get eth, understand gas fees, etc. This is poor UX because of the large learning curve users will have to endure. 
+It is important to note that the architecture of this application is different from that of traditional dApps, which require users to obtain ETH, use a browser wallet, understand gas fees, etc. This presents a steep learning curve for new users and a poor UX. 
 
-With Create-Unirep-App, the architecture is Web App <-> Relay <-> Blockchain which reduces the cognitive load of a user new to blockchain appplications by abstracting the learning curve into the relay:
+With `create-unirep-app`, the architecture is Web App <-> Relay <-> Blockchain. This reduces the cognitive load for a user new to blockchain appplications by abstracting that learning curve into the relay:
 
 ![Architecture](https://imgur.com/VdmLjZe.png)
 
+While use of the relayer is optional for a Unirep application, developers should be aware that if users send transactions directly to `UnirepApp.sol` directly, their privacy could be compromised.
+:::
 
-## 🕹️ Open frontend to interact with the application
-
-### User Sign up
+## 🕹️ Interacting with the example attester
 
-Click on the **Join** button.
+### User Sign Up
 
 ![Join](https://i.imgur.com/vcR3x9D.png)
 
-'Join' will prompt the client to generate a [semaphore identity](https://semaphore.appliedzkp.org/) and a [signup proof](../circuits-api/circuits.md#signup-proof). <br/>
-The relayer will submit the signup proof to `UnirepApp.sol` and the user will keep the semaphore identity secretly in the browser.
-After the transaction successfully submitted, the window will show a **Dashboard** button.
+Clicking the **Join** button prompts the client to generate a [Semaphore Identity](https://semaphore.appliedzkp.org/) and a [signup proof](../circuits-api/circuits.md#signup-proof). <br/>
+The relayer will submit the signup proof to `UnirepApp.sol` and the client will store the Semaphore Identity secretly in the browser.
+When the transaction completes successfully, the user has been added to the attester's membership and can access the application with the  **Dashboard** button.
 
 ![Dashboard](https://i.imgur.com/NURZkvA.png)
 
 ### Information
 
 <a href="https://imgur.com/8922mih"><img src="https://i.imgur.com/8922mih.png" title="Information" width="300"/></a>
```

Comment:
as above with the screenshots - these could be smaller so they don't create so much extra space to scroll through.
i think it would be better if we remove the headings right before the screenshots  (Information, Request Data, etc) bc it is repetitive -or- not use the screenshots.. if the reader is looking at the demo i think it will be easy enough to follow along without them.

Reply:
The **demo** means if they are trying this website?
yeah or just using the description to describe these columns work?

Reply:
yeah, i mean if they are following the directions in Get Started doc and doing the build, joining, looking at the dashboard: with the detailed description in the doc maybe the screenshots are unnecessary.